tests/Microsoft.Fx.Portability.Offline.Tests/OffLineApiPortServiceTests.cs

```diff
@@ -29,23 +28,25 @@ public class OffLineApiPortServiceTests
             CreateApiRecommendations()
         );
 
-        [TestMethod]
-        [ExpectedException(typeof(ArgumentNullException))]
+        [Fact]
         public void QueryDocIdsWithNullThrowsArgumentNullException()
         {
-            var result = _OfflineApiPortService.QueryDocIdsAsync(null).Result;
+            var result = Record.Exception(
+               ()=> _OfflineApiPortService.QueryDocIdsAsync(null).Result
+               );
+            Assert.IsType<ArgumentNullException>(result);
         }
 
-        [TestMethod]
+        [Fact]
         public void QueryDocIdsWithEmptyListOfValidIds()
         {
             var expectedDocIds = new List<string>();
 
             var result = _OfflineApiPortService.QueryDocIdsAsync(expectedDocIds).Result.Response;
```

Comment:
Xunit supports async tests, so removing the .Result with await would be awesome as well.